src/transformState.js

```diff
@@ -6,6 +6,26 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
+        for (const del in state) {
+          delete state[del];
+        }
+        break;
+
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete state[key];
+        }
```

Comment:
Don't forget to add the default case to avoid possible errors when action.type does not match any of the cases

Reply:
fix this comment, please